Formula/fd.rb

```diff
@@ -16,6 +17,13 @@ class Fd < Formula
   def install
     system "cargo", "build", "--release"
     bin.install "target/release/fd"
+
+    # Completions are written into a hashed subdirectory
+    Pathname.glob(buildpath/"target/release/build/fd-find-*/out") do |completion_out|
```

Comment:
Can we ask upstream to have an install script to do that? Like make-based builds have `make install`, to avoid depend too much on details of the build process.

Reply:
We could, but my (limited) understanding is that Rust programs traditionally don't have install scripts. Additionally, [there is precedent for doing this](https://github.com/Homebrew/homebrew-core/blob/74e296a95a0b3cdaf148d3258d1468a8724feab0/Formula/ripgrep.rb#L23-L25).

Reply:
Upstream simplified this for us. See
https://github.com/sharkdp/fd/pull/124
https://github.com/sharkdp/fd/issues/121